apps/server/src/shared/infra/identity-management/keycloak/uc/Keycloak-management.uc.ts

```diff
@@ -0,0 +1,73 @@
+import { Injectable } from '@nestjs/common';
+import fs from 'node:fs/promises';
+import { IJsonAccount, IJsonUser } from '../interface';
+import { KeycloakAdministrationService } from '../keycloak-administration.service';
+
+@Injectable()
+export class KeycloakManagementUc {
+	constructor(private readonly kcAdmin: KeycloakAdministrationService) {}
+
+	public async check(): Promise<boolean> {
+		return this.kcAdmin.testKcConnection();
+	}
+
+	public async clean(): Promise<number> {
+		let kc = await this.kcAdmin.callKcAdminClient();
+		const users = (await kc.users.find()).filter(
+			(user) => user.id !== undefined && user.username !== this.kcAdmin.getAdminUser()
+		);
+
+		// eslint-disable-next-line no-restricted-syntax
+		for (const user of users) {
+			// eslint-disable-next-line no-await-in-loop
+			kc = await this.kcAdmin.callKcAdminClient();
```

Comment:
Would be nicer to use chunks of x users in a `Promise.all` (if keycloak can handle those parallel requests) to speed this up.

Reply:
I tried using `Promise.all`, but got an internal server error as response. An other way to do this is to instantiate the Keycloak Client for every user and than run the create in parallel. But I think this might be overkill for initialization of 60-70 users.